packages/perspective-viewer-d3fc/build.js

```diff
@@ -93,6 +95,19 @@ async function compile_css() {
 }
 
 async function build_all() {
+    // esbuild can handle typescript files, and strips out types from the output,
+    // but it is unable to check types, so we must run tsc as a separate step.
+    try {
+        await execSync("tsc", { stdio: "inherit" });
+    } catch (error) {
+        console.error(error);
+        // tsc errors tend to get buried when running multiple package builds. If
+        // the perspective-viewer-d3fc build fails, then plugins will not be present
+        // when running tests, leading to a large number of tests failing, but without
+        // a great indication of why.
```

Comment:
Ok but ...then why catch the error? Printing and exiting with code 1 is the _default_ behavior for an unhandled exception in this function. This only sticks out to me because we do this [verbatim in `perspecitve-viewer`](https://github.com/finos/perspective/blob/master/rust/perspective-viewer/build.js#L83) without issue.